bin/acfg1.py

```diff
@@ -178,9 +210,11 @@ def get_or_create_ids(username, groupname):
     parser.add_argument("source", help="Source location overlay directory")
     parser.add_argument("destination", help="Destination of where to overlay source directory over")
     parser.add_argument('--dry', action='store_const', const=True, default=False, help='Dry run only')
+    parser.add_argument('--hostclass', action='append', dest='hostclasses', metavar='HOSTCLASS',
+                        help='A hostclass (possibly virtual) for which to select files. (Repeatable.)')
```

Comment:
Sorry, clobbered english. I meant, to pick up hostclass from file in this location (in arg parsing code) as opposed deeper in code (as its done now).